cypress/support/component-helper/common-steps.js

```diff
@@ -23,11 +23,28 @@ export const parseToIntElement = (elem) => {
   return parseInt(inputString);
 };
 
-export const checkGoldenOutline = (elem, outlineWidthPx = 3) => {
-  const outlineWidth = parseToIntElement(elem.css("outline-width"));
-  expect(elem.css("outline-color")).to.equals("rgb(255, 181, 0)");
-  expect(elem.css("outline-style")).to.equals("solid");
-  expect(outlineWidth).to.be.within(outlineWidthPx - 1, outlineWidth + 1);
+export const checkGoldenOutline = (
+  elem,
+  outlineWidthPx = 3,
+  outline = "outline"
+) => {
+  const outlineWidth = parseToIntElement(elem.css(`${outline}-width`));
+  expect(elem.css(`${outline}-color`)).to.equals("rgb(255, 181, 0)");
+  expect(elem.css(`${outline}-style`)).to.equals("solid");
+  expect(outlineWidth).to.be.within(outlineWidthPx - 1, outlineWidthPx + 1);
+};
+
+export const checkOutlineCss = (
```

Comment:
Similar comment wrt naming it something more generic. Neither is essential though, happy to keep as is if you prefer it current naming

Reply:
same here, this checks only the outline (border) for different components with diff styling / colors / width - so I think it is ok as well :)